samza-shell/src/main/bash/run-class.sh

```diff
@@ -154,9 +154,6 @@ fi
 # Check if a max-heap size is specified. If not - set a 768M heap
 [[ $JAVA_OPTS != *-Xmx* ]] && JAVA_OPTS="$JAVA_OPTS -Xmx768M"
 
-# Check if the GC related flags are specified. If not - add the respective flags to JVM_OPTS.
-[[ $JAVA_OPTS != *PrintGCDateStamps* && $JAVA_OPTS != *-Xloggc* ]] && JAVA_OPTS="$JAVA_OPTS -XX:+PrintGCDateStamps -Xloggc:$SAMZA_LOG_DIR/gc.log"
```

Comment:
Does default GC flag for `loggc` configured to generate `gc.log` in the right directory?